sass/sass_wrapper.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * A Sass compiler wrapper that supports bazel persistent worker protocol.
+ * 
+ * Bazel can spawn a persistent worker process that handles multiple invocations. It
+ * will also be backwards compatible with being run as a normal single-invocation command.
+ */
+"use strict";
+const {debug, runAsWorker, runWorkerLoop} = require('@bazel/worker');
+const sass = require('sass');
+const fs = require('fs');
+
+// Equivalent of running node with --expose-gc
+// but easier to write tooling since we don't need to inject that arg to
+// nodejs_binary
+if (typeof global.gc !== 'function') {
+  require('v8').setFlagsFromString('--expose_gc');
+  global.gc = require('vm').runInNewContext('gc');
+}
+
+function main(args) {
+  if (runAsWorker(args)) {
+    debug('Starting Sass compiler persistent worker...');
+    runWorkerLoop(runOneBuild);
```

Comment:
I'd just inline the call to `sass.run_()` here as well.

Reply:
Done.